src/lib/arrays.js

```diff
@@ -0,0 +1,86 @@
+const getNthElement = (index, array) => {
+  if (array > array.length - 1) {
+    index -= array.length;
+    return array[index];
+  }
+  return array[index];
+};
+
+const arrayToCSVString = array => {
+  return array.join(',');
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  return array.concat(element);
+};
+
+const removeNthElement = (index, array) => {
+  return array.splice(index, 1);
+};
+
+const numbersToStrings = numbers => {
+  return numbers.map(String);
+};
+
+const uppercaseWordsInArray = strings => {
+  return strings.map(item => item.toUpperCase());
+};
+
+const reverseWordsInArray = strings => {
+  return strings.map(word =>
+    word
+      .split('')
+      .reverse()
+      .join(''),
+  );
+};
+
+const onlyEven = numbers => {
+  return numbers.filter(a => numbers.indexOf(a) % -2 === 1);
```

Comment:
You were so close here! Would be a good idea to read up a bit more about the how the filter method works just so its good and solid for you. In this case you could do this:
```
  return numbers.filter(number => number % 2 === 0);
```